modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -55,84 +54,46 @@ When you create this API in API Manager:
 
 * Select `HTTP API` as the asset type.
 * Leave the defaults on the *Endpoint configuration* screen.
-* Copy the API instance ID and paste it into the *API Auto discovery id* field in the Partner Manager *New endpoint: Receive from Partners* window.
+* Copy the API instance ID and paste it into the  *API Auto discovery id* field on the Partner Manager *New endpoint: Receive from Partners* window.
```

Comment:
I think "in a window" sounds better than "on a window," but I am still trying to figure out what guidance to give re. use of prepositions with UI elements....